tests/integration/test_local_storage/test.py

```diff
@@ -0,0 +1,148 @@
+import logging
+import os
+import random
+import string
+
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+from pathlib import Path
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
```

Comment:
Could it be a functional test?

Reply:
I don't like writing bash scripts, tbh)

Reply:
Oh, implementing a functional test using python is possible, will try to do it